projects/1.16/assets/akisephilas-bosses/akisephilas_bosses/lang/zh_cn.json

```
@@ -0,0 +1,191 @@
+{
+  "advancements.adorn_me_in_purple.descr": "装备一整套康明“石”护甲",
```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Wear a full set of Culminite Armor**

```suggestion
  "advancements.adorn_me_in_purple.descr": "装备一整套康明“石”护甲",
```

---

大概率是不标准的音译，译名更换为DeepL提供的“烛石”

---

所以可以解释一下这类矿石在游戏中是怎么获取的吗？
机翻不一定准确